### DISCUSSION TITLE
如何在原创页和承诺书添加日期，如2021年6月2日

### DISCUSSION THREAD

**Discussion by YZ-SJTU** [sjtug/SJTUThesis#675] [提问] [answered]
如何在原创页和授权书添加日期，如2021年6月2日？

**Comment by skyzh** [comment, marked as answer]
如果您在使用 master 分支上的版本，授权书 LaTeX 内不太好更改，是直接嵌入 pdf 实现的。<del>如果是 develop 分支上的版本，可以在 `orig@body` 和 `copyrightpage` 宏中修改相关的内容。</del>

**Reply by skyzh** [reply]
去掉 `[xxx.pdf]` 后在 `sjtu@signbox` 中可以直接嵌入日期。

**Reply by YZ-SJTU** [reply]
还有一个问题，请问如何在方框里打钩 ~\sjtu@square和$\checkmark$

**Reply by skyzh** [reply]
在 `\newcommand\sjtu@square{\symbol{"25A1}}` 之后加入
`\newcommand\sjtu@checkedsquare{\text{\rlap{$\checkmark$}}\symbol{"25A1}}`

而后将 `\sjtu@square` 换成 `\sjtu@checkedsquare`
